.env.example

```diff
@@ -21,6 +21,7 @@ VAULT_TOKEN=
 ES_HOST=http://localhost:9200
 ES_USER=''
 ES_PASS=''
+ES_INDEX=''
```

Comment:
this value is given by config files... `/configs/base.json`

we don't want to encourage anyone using ENV for it, because it'll be overwritten